inspire/modules/records/receivers.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2015 CERN.
+#
+# INSPIRE is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+from invenio.ext.sqlalchemy import db
+
+from invenio_records.models import Record
+
+from invenio_records.signals import before_record_insert
+
+
+@before_record_insert.connect
```

Comment:
Hi @Panos512. Discussing IRL with @jalavik, we came to the conclusion the best fit for hooking up this signal is indeed upon loading of INSPIRE demo records, while disable it otherwise.

For this reason a generic `inveniomanage inspire` (or better name) is going to be created such that we can have e.g.:

`$ inveniomanage inspire demorecords foo.xml` 

In the `demorecords` command, and only there, we can then register the signal.

For this reason, we can pause for a second this PR, to define better the above command, so that you can then hook in the proper place your hack :smile: .